docs/gettinginvolved/infrastructure/documentation.rst

```diff
@@ -0,0 +1,112 @@
+Documentation Hosting
+=====================
+
+Flocker's documentation is hosted on S3, fronted by cloudfront and cloudflare.
+
+S3 Buckets
+----------
+
+There are 3 S3 buckets used for documentation.
+
+clusterhq-docs
+~~~~~~~~~~~~~~
+
+This bucket hosts our public facing documentation.
+
+It has documentation for all marketing releases.
+
+Configuration
+`````````````
+It is configured to allow static website hosting, with an index document of ``index.html``.
+
+It has the following bucket policy configured
```

Comment:
Maybe mention why it has this.  `Id` pretty much explains it but I guess it wouldn't hurt to say "to grant anonymous read-only access to all contents" or something.